### DISCUSSION TITLE
Installer Crash v3.1.1 for Catalina

### DISCUSSION THREAD

**Discussion by yuchong-li** [osy86/HaC-Mini#931] [Q&A] [answered]
Hi guys,

I was trying to install the latest Catalina on my NUC8i7HVK4.

I was able to find the boot picker page via shell command.

And when the installer running it will crash and quit at the 3/4 finished position and the log showing like below. (This is the last log before quit) Then I will be redirected to the boot picker page and start the installer again and again. (And it will crash every single time)
![image](https://user-images.githubusercontent.com/49866477/134347024-c611a808-c53c-4b4d-971f-7cde0223dcb3.jpeg)
Any idea from you guys?

SSD is 970 EVO PLUS 1T.

![image](https://user-images.githubusercontent.com/49866477/134348336-b27d06f2-0999-4556-9177-c31bcde518cf.jpeg)
![image](https://user-images.githubusercontent.com/49866477/134354762-6273ea62-9c41-481b-a379-6b5df91a6325.jpeg)
Appreciate for your help~

Best,
Yuchong

**Comment by yuchong-li** [comment, marked as answer]
Ok finally it's solved.

Please do not erase the internal SSD by using APFS but using the default Mac OS Extended (Journaled) format.

And the installation runs smooth.

**Reply by AppleTechy** [reply]
I currently have 2 x 970 Evo Plus drives installed in my NUC. I experienced a similar issue when trying to format a Samsung 970 Evo Plus to AFPS and MacOS journaled when using the MacOS Big Sur installer were it would just cause the system to hang and require a reboot. I noticed on some other [forums](https://forums.macrumors.com/threads/samsung-m-2-970-evo-plus-firmware-upgrade-problem-solved.2220069/), similarly it is mentioned in OpenCore's [documentation](https://dortania.github.io/OpenCore-Install-Guide/macos-limits.html#storage-support),  that Samsung released a later firmware version that fixed compatibility issues with MacOS. I tried using the Samsung provided Linux iso to do the upgrade but it wasn't able to detect any supported drives. I ended up installing Windows temporarily to use the [Samsung Magician software](https://www.samsung.com/semiconductor/minisite/ssd/download/tools/) to successfully upgrade the firmware. Once the drives firmware was upgraded I was able to format as AFPS or MacOS Journaled with no problems.
